package.json

```diff
@@ -93,12 +101,12 @@
     "output-file-sync": "^1.1.1",
     "phantomjs-prebuilt": "^2.1.7",
     "portfinder": "^1.0.3",
-    "react": "^15.0.0",
-    "react-component-metadata": "^2.1.1",
-    "react-dom": "^15.0.0",
+    "react": "^15.0.1",
+    "react-component-metadata": "^3.0.0",
```

Comment:
It looks like you have the latest major version here, that should work just fine with Babel6